src/statick_tooling/plugins/tool/dockerfilelint_tool_plugin.py

```diff
@@ -112,5 +112,16 @@ def parse_output(self, total_output: List[str]) -> List[Issue]:
 
                     except ValueError as ex:
                         logging.warning("ValueError: %s, line: %s", ex, line)
```

Comment:
Same as other comment. Want to keep or remove this logging?

Reply:
I also think the log statements are duplicative. I've removed them.